docs/resources/artifactory_user.md

```diff
@@ -27,14 +23,14 @@ resource "artifactory_user" "test-user" {
 
 The following arguments are supported:
 
-* `name` - (Required) Username for user
-* `email` - (Required) Email for user
-* `password` - (Optional) Password for the user
-* `admin` - (Optional) 
+* `name` - (Required) Username for user.
+* `email` - (Required) Email for user.
+* `admin` - (Optional) When enabled, this user is an administrator with all the ensuing privileges.
 * `profile_updatable` - (Optional) When set, this user can update his profile details (except for the password. Only an administrator can update the password).
 * `disable_ui_access` - (Optional) When set, this user can only access Artifactory through the REST API. This option cannot be set if the user has Admin privileges.
 * `internal_password_disabled` - (Optional) When set, disables the fallback of using an internal password when external authentication (such as LDAP) is enabled.
-* `groups` - (Optional) List of groups this user is a part of
+* `groups` - (Optional) List of groups this user is a part of.
+* `password` - (Required) Password for the user. Password validation is not done by the provider and is offloaded onto the Artifactory side. There may be cases in which you want to leave this unset to prevent users from updating their profile. For example, a departmental user with a single password shared between all department members.
```

Comment:
Why is this moved to the bottom of this group? Same comments from the README applies here.

Reply:
to match the order in the schema.